tests/mobc.rs

```diff
@@ -706,14 +706,6 @@ fn test_max_idle_lifetime() {
         drop(v);
         delay_for(Duration::from_millis(800)).await;
 
-        let mut v = vec![];
```

Comment:
This assertion was incorrect - connection lifeteme is 1 sec, by that time 1800ms passed but connections were not freed for some reason